kv/bucket.go

```diff
@@ -327,24 +323,45 @@ func (s *Service) FindBuckets(ctx context.Context, filter influxdb.BucketFilter,
 		if err != nil {
 			return err
 		}
+
 		bs = bkts
+
 		return nil
 	})
 
-	if err != nil {
-		return nil, 0, err
+	needsSystemBuckets := true
+	for _, b := range bs {
+		if b.Type == influxdb.BucketTypeSystem {
+			needsSystemBuckets = false
+			break
+		}
 	}
 
-	tasks, error := s.findSystemBucket("_tasks")
-	if error != nil {
-		return bs, 0, error
+	if needsSystemBuckets {
+		tb := &influxdb.Bucket{
```

Comment:
Since `tb` and `mb` aren't used anywhere aside from the `append` I suppose you could do:

```go
bs = append(bs, &influxdb.Bucket{
...
})
```

Reply:
I kinda find it more readable the way it is